src/dbt_score/cli.py

```diff
@@ -102,14 +118,35 @@ def lint(
         config.overload({"rule_namespaces": namespace})
     if disabled_rule:
         config.overload({"disabled_rules": disabled_rule})
+    if fail_project_under:
+        config.overload({"fail_project_under": fail_project_under})
+    if fail_any_model_under:
+        config.overload({"fail_any_model_under": fail_any_model_under})
 
     if run_dbt_parse:
         dbt_parse()
 
     try:
-        lint_dbt_project(
+        evaluation = lint_dbt_project(
             manifest_path=manifest, config=config, format=format, select=select
         )
+
+        failed_models = {
+            model.unique_id: score.value
+            for model, score in evaluation.scores.items()
+            if (score.value < config.fail_any_model_under)
+        }
+
+        if failed_models:
+            print(f"Error: fail_any_model_under = {config.fail_any_model_under}")
+            for model, value in failed_models.items():
+                print(f"Model {model} scored {round(value,1)}")
+            ctx.exit(1)
+
+        if evaluation.project_score.value < config.fail_project_under:
+            print(f"Error: fail_project_under = {evaluation.project_score.value}")
+            ctx.exit(1)
```

Comment:
I think it'd make sense to have this output done by `src/dbt_score/formatters/human_readable_formatter.py`. The `Formatter` class doesn't know about the config,  but it could easily receive it upon instantiation in `__init__`. I'm happy to take care of this if you prefer - let me know.

Reply:
@matthieucan I've pushed the other PR fixes but happy for you to fix this. I'm struggling for time this week

Reply:
@WobblyRobbly Unfortunately it seems I can't push to this PR, despite the "Maintainers are allowed to edit this pull request" option. Is there some branch protection on your side?

Reply:
Thanks, that worked!